dot/sync/bootstrap_syncer_integration_test.go

```diff
@@ -28,9 +28,10 @@ func newTestBootstrapSyncer(t *testing.T) *bootstrapSyncer {
 		trie.EmptyHash, 200, types.NewDigest())
 	require.NoError(t, err)
 
-	bs := new(syncmocks.BlockState)
-	bs.On("BestBlockHeader").Return(header, nil)
-	bs.On("GetHighestFinalisedHeader").Return(finHeader, nil)
+	ctrl := gomock.NewController(t)
+	bs := NewMockBlockState(ctrl)
+	bs.EXPECT().BestBlockHeader().Return(header, nil).AnyTimes()
+	bs.EXPECT().GetHighestFinalisedHeader().Return(finHeader, nil).AnyTimes()
```

Comment:
Try to find the number of times each get called.
I'm gonna sound like a madman, but I usually try with `.Times(1)`, then 2, then 3 and that usually solves it 🤣